src/__init__.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+
+from Animals.Bunny import Bunny
+from Animals.Turtle import Turtle
+from Animals.CompetingAnimal import CompetingAnimal
+from Road import Road
+
+havi = CompetingAnimal("Havi the bunny", 0, 3, 'B')
+tirza = CompetingAnimal("Tirza the turtle", 0, 1, 'T')
+all_animals = [havi, tirza]
+race_road = Road(20)
+race_length = race_road.length_in_steps
+
+
+def race_is_over(animals: list):
+    for animal in all_animals:
+        if animal.has_won(race_length):
+            return True
+    return False
+
+
+def print_result(animals: list):
```

Comment:
nice one on the strong typing of the arguments, read how you can strong type with Pydantic, it's better and more useful.
also, you should also strong type what the method return, no?